### PR TITLE
fix(trading): show user inactive lose streak in remaining epochs

### LINKED ISSUE #5590: Rewards page UX tweaks
- Make your current tier in your activity streak or hoarder bonus more evident (bolder/ larger, or we could fade out the streaks that are not applied slightly
image (10)
- In activity and hoarder bonus tiles have the active trader text dynamically change between active and inactive. e.g. "Inactive trader: 4 epochs so far, you will lose your streak benefits in 3 epochs!" 
- Value taken from the inactiveStreak on the API and compairing to the network param `rewards.activityStreak.inactivityLimit`)

image (9)

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes #5590 

# Description ℹ️

- [x] In the activity and hoarder bonus tiles, the active trader text dynamically changes between active and inactive. e.g. "Inactive trader: 4 epochs so far, you will lose your streak benefits in 3 epochs!"
- [x] Value taken from the inactiveStreak on the API and compared to the network param `rewards.activityStreak.inactivityLimit`)

# Demo 📺

Gif/video/images of new/corrected functionality if applicable

# Technical 👨‍🔧

Details of technical implementation that reviewers may need to be aware of, if applicable.

### CODE REVIEW

**Review by asiaznik — APPROVED** [review]

File: apps/trading/components/rewards-container/streaks/activity-streaks.spec.tsx [hunk 1]
```diff
@@ -1,6 +1,17 @@
 import { render, screen } from '@testing-library/react';
 import { ActivityStreak } from './activity-streaks';
 
+jest.mock('@vegaprotocol/network-parameters', () => ({
+  ...jest.requireActual('@vegaprotocol/network-parameters'),
+  useNetworkParams: jest.fn(() => ({
+    params: {
+      rewards_activityStreak_inactivityLimit: '3.00',
```
Comment (bwallacee) [inline apps/trading/components/rewards-container/streaks/activity-streaks.spec.tsx:8]: I don't think 3.00 is valid?
At least in my testing it doesn't get accepted.

**Comment by bwallacee** [comment]
@MadalinaRaicu I dont think this displays correctly:
image
Here I have been active for 1 epoch with inactivity limit set to 3:
```
vega_instance.update_network_parameter(
        MM_WALLET.name, parameter="rewards.activityStreak.inactivityLimit", new_value="3"
    )
```

File: apps/trading/components/rewards-container/streaks/activity-streaks.spec.tsx (hunk 1 above)
Comment (MadalinaRaicu) [reply apps/trading/components/rewards-container/streaks/activity-streaks.spec.tsx:8]: i got it from https://explorer.vega.xyz/network-parameters 
it should be, right?
image

Comment (MadalinaRaicu) [inline apps/trading/components/rewards-container/streaks/activity-streaks.spec.tsx:8]:
```suggestion
      rewards_activityStreak_inactivityLimit: '3',
```

Comment (MadalinaRaicu) [inline apps/trading/components/rewards-container/streaks/activity-streaks.spec.tsx]:
```suggestion
      rewards_activityStreak_inactivityLimit: '3',
```

**Comment by MadalinaRaicu** [comment]
> @MadalinaRaicu I don't think this displays correctly: 

this should be now fixed

**Review by bwallacee — APPROVED** [review]

**Pull request merged**